delft3dworker/management/commands/containersync_sceneupdate.py

```diff
@@ -15,4 +19,48 @@ class Command(BaseCommand):
     help = "sync containers with container and scene model"
 
     def handle(self, *args, **options):
-        self.stdout.write("containersync_sceneupdate skeleton")
+
+        # Loop over non empty celery_task_ids in containers
+        # celery_set = set(
+            # Container.objects.exclude(celery_id__exact='').values_list('celery_id', flat=True))
```

Comment:
Please do not leave commented code without explanation of why it's commented. So either:
- remove completely
- add explanation why you have this in the file
- activate code by removing hashes